### ISSUE TITLE
Automatically apply the "wg-css" label

### ISSUE THREAD

**Issue by frivoal** [w3c/web-platform-tests#10707]
There's a wg-css label, presumably to label tests that cover specs produced by the css working group.

I believe that at some point, this label was automatically applied to any pull request that contains tests  under the css directory, but either that broke down at some point, or it never worked and I just imagined it.

Either way, it would be good to fix that, as that's a useful label, and it would be more useful if it was accurate.

**Comment by frivoal** [comment]
Or maybe it works, but not reliably. Here's a PR where this had to be added manually: https://github.com/w3c/web-platform-tests/pull/10700

**Comment by gsnedders** [comment]
@frivoal The labelling is done by https://github.com/tobie/ganesh. The fundamental problem is that `tr.rdf` doesn't consistently include versionless URLs (it mostly does for non-CSS specs, oddly) and it uses that data to attribute directories to WGs; there's some bunch of emails on www-archive somewhere about that omission.

cc/ @tobie

**Comment by tobie** [comment]
Yeah. Version/edition numbers at W3C are inconsistent even within each WG. Little we can do here except completely rethink Specref to handle it automatically (which no one seems ready to fund) or fix those manually by adding an alias to Specref. I recommend the latter.

**Comment by frivoal** [comment]
csswg has standardized to used unversioned shortnames for test directories. Does that mean we just need to add an unversionned alias to every versioned shortname in specref to workaround the problem?

**Comment by tobie** [comment]
Yes.
